core/invoker/src/main/scala/org/apache/openwhisk/core/containerpool/docker/DockerClient.scala

```diff
@@ -98,6 +99,20 @@ class DockerClient(dockerHost: Option[String] = None,
     Seq(dockerBin) ++ host
   }
 
+  // Determine the version of the docker client cli.
+  // Failure to determine the client version is a fatal error, because it indicates a high probability
+  // that subsequent docker cli commands will also fail.
+  protected def getClientVersion(): String = {
```

Comment:
I suggest to be more explicit here and state that `getClientVersion()` will throw an exception on failure.

Reply:
done